components/automate-deployment/pkg/assets/assets.bindata.go

```diff
@@ -1,19 +1,19 @@
-// Code generated by go-bindata. DO NOT EDIT.
+// Code generated by go-bindata.
```

Comment:
is this correct? is it possible your go bindata is old?

Reply:
It is indeed old. Regenerated.